WBooks/TableBooks/CellBook/CellBookView.swift

```diff
@@ -28,7 +28,21 @@ final class CellBookView: UITableViewCell {
     func configureCell(with viewModel: CellBookViewModel) {
         titleBook.text = viewModel.title
         subtitleBook.text = viewModel.autor
-        imageBook.image = UIImage(named: viewModel.image)
+        
+        if let url = URL(string: viewModel.image) {
+            DispatchQueue.global().async {
+                if let data = try? Data(contentsOf: url) {
+                    DispatchQueue.main.async {
+                        self.imageBook.image = UIImage(data: data)
+                    }
+                }
+            }
+        }
```

Comment:
Looks good, but let's create an extension of UIImageView so that we can reuse it in the future. It would look like this:

```Swift
extension UIImageView {

    func load(from url: String) {
        ...
    }
}
```

Also, it would be located in a different file.